### ISSUE TITLE
Tried to open anaconda and it reported this message. 

### ISSUE THREAD

**Issue by gakuen34590** [conda/conda#12845]
### Checklist

- [X] I added a descriptive title
- [X] I searched open reports and couldn't find a duplicate

### What happened?

![help](https://github.com/conda/conda/assets/138404092/c91c8ff7-d89b-43a4-aa3b-f583872e13c3)
tried to open anaconda and i got this issue. 
i also have the issue where if i try to open anaconda navigator, a black window pops up and closes every 5-10 seconds.

**Comment by rillusions** [comment]
i had the same problem!
and the black page is command prompt
but after a few more attempts it opened!

**Comment by gakuen34590** [comment]
yeah i guess this is an issue for everyone who downloads anaconda. 

fortunately it does go away on its own and theres no other issues with anaconda😃

**Comment by travishathaway** [comment]
Hi all,

Thanks for creating this issue. The appropriate place for it is over at this project's issue queue:

- https://github.com/ContinuumIO/anaconda-issues/issues

Therefore, I will be closing this issue for now. Please re-open in the project above if you believe it still needs attention.

One more thing I would like to mention is that Anaconda currently has an ongoing public beta testing for the "next generation" navigator. More information on that can be found here: https://www.anaconda.com/blog/anaconda-navigator-the-next-generation
